core/server/proxy/src/main/java/alluxio/proxy/s3/ListBucketResult.java

```diff
@@ -63,7 +63,7 @@
   private List<Content> mContents;
 
   // List of common prefixes (aka. folders)
-  private CommonPrefixes mCommonPrefixes;
+  private List<CommonPrefixes> mCommonPrefixes;
```

Comment:
Also change the comment here?

Reply:
Comment `List of common prefixes` is right.  I don't think it needs to be modified.

Like comment of `mContents`:
```
  // List of files.
  private List<Content> mContents;
```